templates/solutions/ubuntu-os/workstations.html

```diff
@@ -0,0 +1,445 @@
+{% block content %}
+  {% set breadcrumbs = [{"name": "Ubuntu OS", "href": "/solutions/ubuntu-os"}, {"name": "Linux Workstations"}] %}
+  {% include '/partial/_breadcrumbs.html' %}
+
+  {%- call(slot) vf_hero(
+    title_text="Take your workstations further with Ubuntu",
+    layout="50/50-full-width-image",
+    is_split_on_medium="true") -%}
+    {%- if slot == 'description' -%}
+      <p>
+        Make your enterprise workstations the pathway to trusted open source. Choose Ubuntu for your enterprise workstations and empower both developers and hardware to achieve peak performance. Benefit from official Ubuntu certification for over 150 laptops and desktops.
+      </p>
+      <p>
+        Give developers access to an ecosystem that puts the latest tooling at their fingertips, backed by enterprise-grade security maintenance from Canonical, the publisher of Ubuntu.
+      </p>
+    {% endif %}
+    {% if slot == "image" %}
+      {{ image(url="https://assets.ubuntu.com/v1/122b651a-hero.jpg",
+            alt="",
+            width="2464",
+            height="1028",
+            hi_def=True,
+            loading="auto") | safe
+      }}
+    {% endif %}
+    {% if slot == "cta" %}
+      <a href="/contact-us"
+         aria-controls="ubuntu-os-modal"
+         class="p-button--positive js-invoke-modal">Contact us</a>
+    {% endif %}
+  {%- endcall -%}
+
+  <section class="p-section">
+    <div class="p-section--shallow">
+      <div class="row">
+
+        <hr class="p-rule" />
+        <h2>
+          Linux workstations
+          <br class="u-hide--medium u-hide--small" />
+          for enterprise-grade deployments
+        </h2>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-start-large-4 col-9">
+        <div class="p-equal-height-row">
+          <div class="p-equal-height-row__col">
+            <div class="p-equal-height-row__item u-hide--small">
+              <div class="p-image-container is-highlighted">
+                {{ image(url="https://assets.ubuntu.com/v1/30b00d6d-no1-amongst-linux-devs.png",
+                                alt="",
+                                width="568",
+                                height="853",
+                                hi_def=True,
+                                loading="lazy") | safe
+                }}
+              </div>
+              <hr class="p-rule--highlight" />
+            </div>
+
+            <div class="p-equal-height-row__item">
+              <h3 class="p-heading--5">#1 amongst Linux developers</h3>
+              <p>
+                A good developer experience boosts productivity. Choose the OS that developers value for its stability, robustness and consistent experience.  Pioneer intensive use cases such as AI, graphics and virtual reality with a software ecosystem that comes with the latest open source tooling.
+              </p>
+            </div>
+            <div class="p-equal-height-row__item">
+              <hr class="is-muted" />
```

Comment:
```suggestion
              <hr class="p-rule--muted" />
```
Vanilla deprecated the `is-muted` classname (see the updated docs [here](https://vanillaframework.io/docs/patterns/rule#muted)). Can you please update all the muted classes with the new classname?